lib/fuzion/std/fileio.fz

```diff
@@ -62,23 +62,15 @@ public fileio is
 
   # retrieves the file size in bytes and returns an outcome of error in case of an error
   #
-  public get_file_size(
-                       # the (relative or absolute) file name, using platform specific path separators
-                       path string) outcome i64 is
-    arr := fuzion.sys.c_string path
-    size := get_file_size arr unit
-    if size != -1
-      size
-    else
-      error "an error occurred while retrieving the size of the file/dir: \"$path\""
-
-  # intrinsic that returns the file size in bytes or -1 in case of an error
+  # NYI: should this use effects?
```

Comment:
I think, yes it should.

Reply:
what is in `fuzion.std.` should be private helpers used by public effects, so this should not use an effect.